invenio_theme/assets/semantic-ui/less/invenio_theme/theme/collections/grid.overrides

```diff
@@ -108,6 +108,8 @@
   }
 
   & > .side-bar.column {
+    display: flex;
+    flex-direction: column;
     min-height: 100vh;
```

Comment:
follow up on what we discussed about the sidebar height
```suggestion
    min-height: calc(100vh - @relative75px);
```

Reply:
Applied! Ty!